packages/x-components/src/views/home/Home.vue

```diff
@@ -615,7 +621,37 @@
       }
     };
 
-    protected queriesToPreview = ['sunglasses', 'handbag', 'earrings', 'jeans', 't-shirt'];
+    protected queriesPreviewInfo: QueryPreviewInfo[] = [
+      {
+        query: 'cortina',
+        extraParams: {
+          instance: 'lolahome',
+          store: 'hola',
```

Comment:
We can use here `store: 'Gijón'` to suit e2e `extra-params.feature` 's `QueryPreviewExtraParamValue`